frontend/packages/console-dynamic-plugin-sdk/src/extensions/create-resource.ts

```diff
@@ -0,0 +1,30 @@
+import { match as RouterMatch } from 'react-router';
+import { ExtensionK8sModel } from '../api/common-types';
+import { CodeRef, Extension, ExtensionDeclaration } from '../types';
+
+export type CreateResource = ExtensionDeclaration<
+  'console.resource/create',
+  {
+    /** The model for which this create resource page will be rendered. */
+    model: ExtensionK8sModel;
+    /** The component to be rendered when the model matches */
+    component: CodeRef<
+      React.ComponentType<{
+        match: RouterMatch<{
+          /** Current namespace */
+          ns?: string;
+          /**
+           * Name of the OLM operator.
+           * Defined if the page is opened from the OLM installed operators page.
+           */
+          appName?: string;
```

Comment:
not new, but `appName` is kind of ambiguous. If we aren't already using "appName" elsewhere in the dynamic plugin API, now would be a good time to change it. It will be hard/impossible to change later.  IMO, it would clearer to call this csvName, operatorName, or something along those lines.

```suggestion
          /**
           * Name of the OLM Operator ClusterServiceVersion that manages this resource.
           * Defined if resource creation is initiated from the Operator details page.
           */
           csvName?: string;
```

Reply:
Thanks @TheRealJon ! I also didnt like the current param name `appName` instead of more obvious like `csvName` but I thought there is a good reason for it - good that you want to change it too.

The latest update has is using `csvName`

Reply:
Yes, now is a good time to change things.